end2end-test-examples/echo-client/README.md

```diff
@@ -6,18 +6,46 @@ This is a simple client for e2e testing against our Echo service defined in [ech
 ## Example
 
 Client sends out `numRpcs` number of unary requests to `host` sequentially
-with request payload size of `reqSize`, and expected response payload size of `rspSize`.
+with request payload size of `reqSize`, and expected response payload size of `rspSize`:
 
-Run test:
+```sh
+./gradlew run --args="--numRpcs=100 --reqSize=100 --resSize=100 --host=grpc-cloudapi1.googleapis.com"
```

Comment:
Why is resSize needed for an echo server if response size would be the same as the request size? Or is there a way to have a larger response size?

Reply:
Actually you can set the response size to be different number. The server will respect that number and generates a repeated string with that size.